common/services/prismic/parsers.js

```diff
@@ -548,7 +549,8 @@ export function parseBody(fragment: PrismicFragment[]): any[] {
         case 'collectionVenue':
           return {
             type: 'collectionVenue',
-            value: slice.primary.content,
+            display: slice.slice_label === 'showClosingTimes',
+            value: parseVenueTimesToOpeningHours(slice.primary.content),
```

Comment:
Should we not just use a `parseCollectionVenue`?
Just wondering, did we not think about making these places?

Reply:
Yeah, I'll change the name of that.
And yes, will probably end up making them places.